docset/windows/smbshare/New-SmbMapping.md

```diff
@@ -233,6 +233,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -UseWriteThrough
+This features was added in Windows Server 2019 and Windows 10 version 1809.
+[UseWriteThrough ensures that writes are completed prior to the write operation returning as completed](https://docs.microsoft.com/en-us/windows-server/storage/file-server/file-server-smb-overview).
```

Comment:
```suggestion
This parameter ensures that writes are completed prior to the write operation returning as completed, for more information, see [Features added in Windows Server 2019 and Windows 10, version 1809](https://docs.microsoft.com/windows-server/storage/file-server/file-server-smb-overview#features-added-in-windows-server-2019-and-windows-10-version-1809).
```

Reply:
@sferrell615

The PR is good to go once the suggested changes have been implemented. Thank you for your patience with the process.